ShoppingList.txt

```diff
@@ -3,4 +3,5 @@ Stuff to buy:
 1. Yoga pad
 2. Stamp pad
 3. Dumbells
-4. Flowers
\ No newline at end of file
+4. Flowers
+5. Napkin
```

Comment:
It is very important to be clean, good job Jeff. Just add screen cleaner as well.